test/e2e/multiple_zk_test.go

```diff
@@ -57,8 +58,9 @@ func testMultiZKCluster(t *testing.T) {
 	defaultCluster.Status.Init()
 	defaultCluster.Spec.Persistence.VolumeReclaimPolicy = "Delete"
 	defaultCluster.ObjectMeta.Name = "zk2"
-	initialVersion := "0.2.5"
-	upgradeVersion := "0.2.7"
+	initialVersion := "0.2.7"
+	upgradeVersion := "0.2.8-rc0"
```

Comment:
Could you look at making the `initial` and `upgrade` Versions configurable so we don;t need to change this file everytime we do a new ZK release?